### ISSUE TITLE
Support for polygon pads?

### ISSUE THREAD

**Issue by evanshultz** [pointhi/kicad-footprint-generator#117]
KiCad v5 adds polygon pads. Will KiCadModTree support them?

The RectFill function produces an approximation, but with many lines and only in rectangular shapes. Even when it works, it's more clunky.

**Comment by pointhi** [comment]
You are right. Supporting polygons would simplify rectangular shapes for example by quite some amount. The Approximation always had the problem of being very prone to changes in line width.

I will look into the support of polygon lines as well as polygon pads if possible soon due to the fact they provide quite some improvement.
